Formula/jinja2-cli.rb

```diff
@@ -0,0 +1,58 @@
+class Jinja2Cli < Formula
+  include Language::Python::Virtualenv
+
+  desc "CLI for the Jinja2 templating language (like a fancier 'envsubst')"
```

Comment:
Could you remove the parenthesis and the information inside it ? It feels opinionated and I'd prefer just the clean description.